pkgs/applications/misc/prusa-slicer/super-slicer.nix

```diff
@@ -0,0 +1,48 @@
+{
+  stdenv, lib, fetchFromGitHub, makeDesktopItem, prusa-slicer
+}:
+let
+  appname = "SuperSlicer";
+  version = "2.2.53.1";
+  pname = "super-slicer";
+  override = super: {
+    inherit version pname;
+
+    src = fetchFromGitHub {
+      owner = "supermerill";
+      repo = "SuperSlicer";
+      hash = "sha256-CAhwmQ63N/XJYToTnIV84lNnjDGNbkmYPzNKNL/wVxs=";
```

Comment:
I think most packages use `sha256 = ...`

Reply:
that's true, but since even "nix-prefetch-github" started throwing those hashes instead on sha256, I just sticked with that, and replaced "sha256 =" with "hash =" to avoid redundancy)